### ISSUE TITLE
root POM is not available in the repositories

### ISSUE THREAD

**Issue by dennisg** [TheThingsNetwork/java-app-sdk#40]
It seems that currently the root POM (`app-sdk`) is not available. Might be due to a rename from `java-app-lib` to `app-sdk` ?

This causes:

```
prompt$ ./gradlew build
:compileJava

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compile'.
> Could not resolve org.thethingsnetwork:data-mqtt:2.0.0.
  Required by:
      nl.dennisg.lora:ttn-client:1.0-SNAPSHOT
   > Could not resolve org.thethingsnetwork:data-mqtt:2.0.0.
      > Could not parse POM https://repo1.maven.org/maven2/org/thethingsnetwork/data-mqtt/2.0.0/data-mqtt-2.0.0.pom
         > Could not resolve org.thethingsnetwork:data:2.0.0.
            > Could not resolve org.thethingsnetwork:data:2.0.0.
               > Could not parse POM https://repo1.maven.org/maven2/org/thethingsnetwork/data/2.0.0/data-2.0.0.pom
                  > Could not find org.thethingsnetwork:app-sdk:2.0.0.
                    Searched in the following locations:
                        https://repo1.maven.org/maven2/org/thethingsnetwork/app-sdk/2.0.0/app-sdk-2.0.0.pom
                        https://repo1.maven.org/maven2/org/thethingsnetwork/app-sdk/2.0.0/app-sdk-2.0.0.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.18 secs
```

Indeed, the mentioned URLs are 404:

![image](https://cloud.githubusercontent.com/assets/243693/21294706/ba94ffd8-c543-11e6-94f0-93558fc4a091.png)

Up to now I just downloaded the jars, but automatic dependency management would be nice ;-)

**Comment by cambierr** [comment]
Well, you're right. This comes from the fact that we did a quick and partial release but will be fixed in 2.1.0 since this release will contains both data amqp, data mqtt, account, and management :)

Since it's already fix in the dev branch, I'm closing this issue ;-)

**Comment by martinbeentjes** [comment]
Hi, any news on a fix for this? I want to work with it, but it is still at 2.0.0 and there is no fix.
